user/managing-os/hvm.md

```diff
@@ -162,7 +192,7 @@ Note how the MAC addresses differ between those two otherwise identical VMs. The
 /.../
 ~~~
 
-If for any reason one would like to make sure that the two VMs have the same MAC address, one can use qvm-prefs to set a fixed MAC address for the VM:
+If for any reason you would like to make sure that the two VMs have the same MAC address, you can use `qvm-prefs` to set a fixed MAC address for the VM:
```

Comment:
This is still possible. And you can actually use this to set different mac addresses too.